driver/pelican/types/pelican.go

```diff
@@ -37,6 +37,8 @@ type Pelican struct {
 	target        string
 	timezone      *time.Location
 	req           *gorequest.SuperAgent
+	drReq         *gorequest.SuperAgent
```

Comment:
`SuperAgent` instances aren't thread safe. I was routinely getting panics during my testing because we now have multiple goroutines for each Pelican thermostat simultaneously issuing requests.

Here, the Pelican structs now have one `SuperAgent` instance for each of their associated goroutines: The main status goroutine, the DR event goroutine, and the new occupancy data goroutine.

I'm open to suggestions if we want to take a different approach.

Reply:
The alternative I guess would be a whole locking wrapper around the `SuperAgent`, which doesn't strike me as worth it. This looks great!